src/main/java/magicbees/bees/BeeSpecies.java

```diff
@@ -186,6 +186,8 @@ public enum BeeSpecies {
             EnumTemperature.NORMAL, EnumHumidity.NORMAL, true, true),
     TC_CHAOS("TCChaos", "tenebrarum", BeeClassification.THAUMIC, 0xCCCCCC, BodyColours.THAUMCRAFT_SHARD,
             EnumTemperature.NORMAL, EnumHumidity.NORMAL, true, false),
+    TC_ESSENTIA("TCEssentia", "defaultium essentia apis", BeeClassification.THAUMIC, 0xCCCCCC,
+            BodyColours.THAUMCRAFT_SHARD, EnumTemperature.NORMAL, EnumHumidity.NORMAL, true, false),
```

Comment:
there's a part of me that wishes it had a unique texture, but eh, no point for it, lotta work when all of the bees practically already look the same.